### PR TITLE
[Bug] LearnMore in Payments is cut-off

### LINKED ISSUE #12241: LearnMore in Payments is cut-off
**Describe the bug**
The learn more section at the bottom of the payments screen is cut off on my Pixel 8 Android 14.

**To Reproduce**
1. Go to More menu
2. Click on Payments
3. Scroll down
4. Notice the last section is cut off by the screen edge.

**Screenshots**

**Expected behavior**

The section is properly rendered on the screen.

### PR DESCRIPTION
**Pull request by backwardstruck**
Closes: #12241  

### Description  

Fixes the issue where the "Learn More" section in the Payments screen is cut off by the screen edge. The issue was caused by improper layout constraints which didn't account for different screen sizes, leading to the content being cut off. 

Also made a text change that had been suggested here:
pecCkj-15h-p2

### Steps to reproduce  

1. Go to the More menu.
2. Click on Payments.
3. Scroll down.
4. Observe that the last section should not be cut off by the screen edge.

#### Devices Used:
- Pixel 8 emulator (Android 14)
- small tablet emulator (Android 13

#### Test Cases:
- Navigating to the Payments screen and scrolling down to verify that the "Learn More" section is fully visible.
- Changing device orientation (portrait/landscape) and checking the "Learn More" section.
- Verifying on different screen sizes, including tablets, that the section is intact.
- Ensuring no other elements on the Payments screen are affected by the layout adjustments.

### The tests that have been performed  

- Verified that the "Learn More" section is fully visible on the Payments screen on multiple devices.
- Ensured that layout changes do not impact other elements on the screen.

- [x] I have considered if this change warrants release notes and have added them to `RELEASE-NOTES.txt` if necessary. Use the "[Internal]" label for non-user-facing changes.

## Reviewer (or Author, in the case of optional code reviews):

Please make sure these conditions are met before approving the PR, or request changes if the PR needs improvement:

- [x] The PR is small and has a clear, single focus, or a valid explanation is provided in the description. If needed, please request to split it into smaller PRs.  
- [x] Ensure Adequate Unit Test Coverage: The changes are reasonably covered by unit tests or an explanation is provided in the PR description.  
- [x] Manual Testing: The author listed all the tests they ran, including smoke tests when needed (e.g., for refactorings). The reviewer confirmed that the PR works as expected on big (tablet) and small (phone) in case of UI changes, and no regressions are added.

### CODE REVIEW

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Message">
          1 Message
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:book:</td>
      <td data-sticky="false">This PR is still a Draft: some checks will be skipped.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

File: WooCommerce/src/main/res/values/strings.xml [hunk 1]
```diff
@@ -3740,7 +3740,7 @@
     <string name="more_menu_button_wc_admin_description">Manage more on admin</string>
 
     <string name="more_menu_button_payments">Payments</string>
-    <string name="more_menu_button_payments_description">Join the mobile payments</string>
+    <string name="more_menu_button_payments_description">Take payments on the go</string>
```
Comment (backwardstruck) [inline WooCommerce/src/main/res/values/strings.xml:3743]: This is an optional change that I threw in.

**Review by kidinov — COMMENTED** [review]
@backwardstruck thank you for your work on this. The solution works well, but I'd propose to take a simpler one. I think the only you need here is to add

```
app:layout_constraintBottom_toBottomOf="parent"
```

to the `RecyclerView` in `fragment_payments_hub`

image

**Comment by dangermattic** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true" data-kind="Warning">
          1 Warning
      </th>
     </tr>
  </thead>
  <tbody>
    <tr>
      <td>:warning:</td>
      <td data-sticky="false">This PR is assigned to the milestone <a href="https://github.com/woocommerce/woocommerce-android/milestone/232">20.1</a>. This milestone is due in less than 2 days.<br />
Please make sure to get it merged by then or assign it to a milestone with a later deadline.</td>
    </tr>
  </tbody>
</table>

<p align="right" data-meta="generated_by_pr-check">
  Generated by :no_entry_sign: <a href="https://danger.systems/">Danger</a>
</p>

**Comment by backwardstruck** [comment]
> @backwardstruck thank you for your work on this. The solution works well, but I'd propose to take a simpler one. I think the only you need here is to add
> 
> ```
> app:layout_constraintBottom_toBottomOf="parent"
> ```
> 
> to the `RecyclerView` in `fragment_payments_hub`
> 
> image

Indeed, the root cause was simply the missing constraint on the RecyclerView, which I initially failed to see. Adding that constraint will indeed fix the display issue. Thank you for this observation!

**Review by kidinov — APPROVED** [review]
LGTM!

**Pull request merged**